pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1067,12 +1067,18 @@ ProfileModel:
       birthYear:
         type: integer
         example: 1970
+        deprecated: true
       birthMonth:
         type: integer
         example: 1
+        deprecated: true
       birthDayInMonth:
         type: integer
         example: 1
+        deprecated: true
+      birthDate:
+        type: string
+        example: 2019-10-19
```

Comment:
We can also add `format: date` here. ([docs](https://swagger.io/docs/specification/data-models/data-types/#string))